### PR TITLE
add packages management service APIs

### PR DESCRIPTION
**Pull request by freeznet**
https://github.com/streamnative/pulsarctl/issues/434

### CODE REVIEW

**Comment by freeznet** [comment]
@zymap @nodece this PR is ready, please help to review this PR when you have time, thanks.

File: pkg/ctl/packages/upload_test.go [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+		"--path", jarName,
+	}
+
+	output, execErr, err := TestPackagesCommands(uploadPackagesCmd, args)
+	failImmediatelyIfErrorNotNil(t, execErr, err)
+	assert.Equal(t, output.String(),
+		fmt.Sprintf("The package '%s' uploaded from path '%s' successfully\n", packageURL, jarName))
+}
+
+func TestUploadPackagesWithFailure(t *testing.T) {
```
Comment (nodece) [inline pkg/ctl/packages/upload_test.go:48]: It same with `TestUploadPackages` test, please check this test.

File: pkg/ctl/packages/upload.go [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+	"github.com/streamnative/pulsarctl/pkg/cmdutils"
+	"github.com/streamnative/pulsarctl/pkg/pulsar/common"
+	"github.com/streamnative/pulsarctl/pkg/pulsar/utils"
+)
+
+func uploadPackagesCmd(vc *cmdutils.VerbCmd) {
+	desc := cmdutils.LongDescription{}
+	desc.CommandUsedFor = "Upload a package."
```
Comment (nodece) [inline pkg/ctl/packages/upload.go:30]: Remove `.`.

**Review by nodece — CHANGES_REQUESTED** [review]

**Comment by freeznet** [comment]
@nodece thanks for the review, have addressed the comments in the latest commits.

**Review by nodece — APPROVED** [review]
LGTM

**Pull request merged**